contracts/DepositContract.sol

```diff
@@ -1,62 +1,27 @@
 pragma solidity >=0.4.22;
 
 contract DepositContract {
-    mapping(address => uint) private Stakes;
-    mapping(address => uint) private Indices;
```

Comment:
Can you explain the reason for these big change on the contract?

Reply:
Those extra data structures were added to enable us to count number of stakers and order in which they staked (e,g `accountList`). This functionality is not required currently. We have this information at block level and keep a running memory of staked accounts. Removed it from the contract to simplify it.

Reply:
It doesn't hurt to keep the information in smart contract. But I won't block you now.

Reply:
Thanks have a issue for myself, if it ever becomes very relevant again: https://github.com/harmony-one/harmony/issues/438